python-lib/tagger_formatter.py

```diff
@@ -0,0 +1,443 @@
+"""
+Module to write the output dataframe depending on the output format
+This module inherits from Tagger where the tokenization, sentence splitting and Matcher instanciation has been done
+"""
+from tagger import Tagger
+from collections import defaultdict
+import pandas as pd, numpy as np
+from enum import Enum
+from typing import List, AnyStr, Dict
+from spacy.tokens import Span, Doc
+
+
+class OutputFormat(Enum):
+    ONE_ROW_PER_DOC = "one_row_per_doc"
+    ONE_ROW_PER_DOC_JSON = "one_row_per_doc_json"
+    ONE_ROW_PER_TAG = "one_row_per_tag"
+
+
+class TaggerFormatter:
```

Comment:
Idea: break down the different output formats in several subclasses? this time with a class inheritance.

I am not sure of the cost/benefit ratio, if you feel like that's overengineering, let me know